### PR TITLE
Allow users to disable the "some projects have trouble loading" popup

### PR DESCRIPTION
**Pull request by rchande**
Fixes https://github.com/OmniSharp/omnisharp-vscode/issues/2110

### CODE REVIEW

**Comment by rchande** [comment]
Thanks @akshita31 for all of your refactorings efforts. This change was super simple for me to implement and test because of your hard work :)

File: package.json [hunk 1]
```diff
@@ -556,6 +556,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.disableMsBuildDiagnosticWarning": {
```
Comment (DustinCampbell) [inline package.json]: It's "MSBuild", not "MsBuild".

Comment (DustinCampbell) [reply package.json]: I note that you didn't take my bit of feedback here @rchande. :smile:

**Comment by rchande** [comment]
@DustinCampbell I intended to (095d86f793c7643fadba3a80188420747d625761) but it looks like I might have missed some files. Fixing...

**Pull request merged**